lib/bridge-client/index.js

```diff
@@ -513,11 +521,9 @@ BridgeClient.prototype._startDemuxing = function(options, cb) {
     }
   );
 
-  var rs = self._store.createReadStream(fileHash);
-
   var uploadState = new UploadState({
     id: id,
-    file: rs,
+    file: file ? file : self._store.createReadStream(fileHash),
```

Comment:
Should we move this up into the logic of the calling function? Seems strange putting it here.